### ISSUE TITLE
Image on card

### ISSUE THREAD

**Issue by andresmazzo** [budparr/gohugo-theme-ananke#217]
In the _recent posts_ of homepage it appears the image, but in the list it doesn't. Would be great if it is possible to configure this. In some types of posts is very important the visual impact. What do you think?
I could contribute in the develop of the improvement.

Thanks for considering!

**Comment by budparr** [comment]
Yes, I agree, that would be good. As long as there's a provision for when there are no images. That is, both on any given entry (so they all look uniform) and for all of them. PRs welcome! Thank you.

**Comment by pdurbin** [comment]
I just experimented with switching the list from "summary.html" to "summary-with-image.html" and I'm pretty happy with the results.

## Before

<img width="1321" alt="Screen Shot 2020-01-04 at 5 49 40 AM" src="https://user-images.githubusercontent.com/21006/71764447-07529680-2eb6-11ea-9b56-7afdfe6cea8b.png">

## After

<img width="1321" alt="Screen Shot 2020-01-04 at 5 49 06 AM" src="https://user-images.githubusercontent.com/21006/71764448-07529680-2eb6-11ea-80cb-5186bef13809.png">

Specifically, this is the change I played with:

```
diff --git a/layouts/_default/list.html b/layouts/_default/list.html
index 130a357..691ec0b 100755
--- a/layouts/_default/list.html
+++ b/layouts/_default/list.html
@@ -6,7 +6,7 @@
     <section class="flex-ns flex-wrap justify-around mt5">
       {{ range .Paginator.Pages }}
         <div class="relative w-100 w-30-l mb4 bg-white">
-          {{- partial "summary.html" . -}}
+          {{- partial "summary-with-image.html" . -}}
         </div>
       {{ end }}
     </section>
diff --git a/layouts/_default/single.html b/layouts/_default/single.html
```
